shim.go

```diff
@@ -96,6 +97,18 @@ func newShimConfig(config PodConfig) interface{} {
 	}
 }
 
+func stopShim(pid int) error {
+	if pid <= 0 {
```

Comment:
Maybe this should be `pid <= 1` :smile: Probably not reasonable for this function to make too many assumptions though.